index.html

```diff
@@ -64,5 +62,199 @@ <h1>
             </section>
         </div>
 
+
+        <section id="work-area-grid">
+
+            <section class="work-card first-card flex">
+                <img class="work-card-desktop-image"
+                    src="images/first-work-card.png" alt="Tonic work card">
+                <img class="work-card-mobile-image"
+                    src="images/first-work-card-mobile.png" alt="Tonic work
+                    card">
+                <div class="work-card-content">
+                    <h1>Tonic</h1>
+                    <ul class="flex work-card-tonic-top-links">
+                        <li>CANOPY</li>
+                        <li class="work-card-top-links-2-3">Back End Dev</li>
+                        <li class="work-card-top-links-2-3">2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads;
+                        no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-card-languages-ul flex">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
```

Comment:
- [ ] [OPTIONAL] Add some padding to the stacks used for each project.
![Screenshot from 2021-08-14 02-31-19](https://user-images.githubusercontent.com/65501535/129427069-b608e188-06a4-434f-b8fa-b60396e4616f.png)